test/unit/adapters/http.js

```diff
@@ -1078,5 +1080,38 @@ describe('supports http with nodejs', function () {
     });
   });
 
+  it('should allow passing FormData', function (done) {
+    var form = new FormData();
+
+    form.append('foo', "bar");
+
+    server = http.createServer(function (req, res) {
+      var form = new formidable.IncomingForm();
+
+      form.parse(req, function (err, fields, files) {
+        if (err) {
+          return done(err);
+        }
+
+        res.end(JSON.stringify({
+          fields: fields,
+          files: files
+        }));
+      });
+    }).listen(4444, function () {
+      axios.post('http://localhost:4444/', form, {
+        headers: {
+          'Content-Type': 'multipart/form-data'
+        }
+      }).then(function (res) {
+        assert.deepStrictEqual(res.data, {
+          fields: {foo: "bar"},
+          files: {}
+        });
+        done();
+      }).catch(done);
```

Comment:
If it gets the the catch clause, will the test fail? As there is no assert I fear the test will pass.

Reply:
Definitely. Passing an argument to the `done` function fails the test with the given error.